src/app/models/Region.ts

```diff
@@ -91,17 +92,24 @@ export class Region
   @deleter()
   public deleter?: User;
 
+  public get license(): Promise<string | null> {
+    // TODO: Use the "project" association once we support async associations
+    // see: https://github.com/QutEcoacoustics/workbench-client/issues/2148
+    const projectsApi = this.injector.get(PROJECT.token);
+    const request$ = projectsApi
+      .show(this.projectId)
+      .pipe(map((project: Project) => project.license ?? null));
+
+    return firstValueFrom(request$);
+  }
+
```

Comment:
See comments above. Would suggest moving todo comment into invocation in the resource